recovery/cmd/launchBackup.go

```diff
@@ -126,6 +134,61 @@ func LaunchBackup() error {
 
 }
 
+// InitBackup Prepare backup pre-requisites
+// returns: 			error
+func InitBackup() error {
+
+	hostDev := filepath.Join(host, "dev")
+	hostDevNull := filepath.Join(hostDev, "null")
+	hostSysroot := filepath.Join(host, "sysroot")
+
+	// Create null device (/dev/null) on /host
+	if err := os.MkdirAll(hostDev, 0o644); err != nil {
```

Comment:
Why 644 for the directory vs 755?

Reply:
The goal was to further restrict the jailed root privileges. I can change to 755.

Reply:
I was just curious, as it seemed unusual for a directory, and didn't match the host /dev permissions. I have no objections, but a comment here would be good to call out the intentional difference

Reply:
Changed to match typical host /dev permissions:
```sh
drwxr-xr-x.   19 root root 3460 Aug 23 02:01 dev
```